tagstudio/src/core/library.py

```diff
@@ -735,17 +740,20 @@ def to_json(self):
 
         file_to_save: JsonLibary = {
             "ts-version": VERSION,
-            "ignored_extensions": [],
+            "ext_list": [],
+            "is_exclude_list": True,
             "tags": [],
             "collations": [],
             "fields": [],
             "macros": [],
             "entries": [],
+            "ignored_extensions": [],
```

Comment:
This fixes the mypy error.
But it also saves an empty list in the json file, which I wanted to avoid.

Reply:
if you will [mark it ](https://github.com/TagStudioDev/TagStudio/pull/251/files#r1629367462)via `NotRequired`, you dont have to set it here
```suggestion

```